3001-4000/LIT3327Melchiz2.xml

```diff
@@ -5,9 +5,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1" xml:lang="en" type="main">Second homily by Cyril of Alexandria on
+           
+            <title xml:lang="gez" xml:id="t1" type="complete" subtype="inscriptio"> ዓዲ፡ ካልእ፡ ድርሳን፡ ዘቅዱስ፡ ቄርሎስ፡ በእንተ፡ መልከ፡ ጼዴቅ፡</title>
```

Comment:
as above, either `en `as `corresp `and `type=main`, or switch `t1 `and `t2`, as it is `t1 `or `type=main` that is printed for `msItem `and elsewhere, and Aaron just now highlighted the importance to refer to uniform titles